presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -734,26 +734,63 @@ Procedures
 
 Use the CALL statement to perform data manipulation or administrative tasks. Procedures are available in the ``system`` schema of the catalog.
 
+CALL statement supports passing arguments by name or by position. Mixing position and named arguments is not supported.
+
+* Named arguments
+
+Procedure arguments all have a name. When passing arguments by name, arguments can be in any order
+and any optional argument can be omitted::
```

Comment:
IMO this should be documented more at the level of the `CALL` syntax rather than in the iceberg connector documentation. We shouldn't need it here as it isn't really iceberg-specific. Same with the positional argument syntax block below.

@steveburnett thoughts?

Reply:
That's a good point @ZacBlanco! If it's not Iceberg-specific and is relevant to all `CALL` statements, then I agree this content should be moved to the [CALL](https://github.com/prestodb/presto/blob/master/presto-docs/src/main/sphinx/sql/call.rst) doc. 

I would think it appropriate to keep the Procedures heading, and the sentence "Use the CALL statement to perform data manipulation or administrative tasks. Procedures are available in the ``system`` schema of the catalog.", and add "See :doc:`/sql/call`." Does that work?

Reply:
+1

Reply:
Even better, delete my added "See :doc:`/sql/call`." and use the link in the sentence itself, like this: 

```
Use the :doc:`/sql/call` statement to perform data manipulation or administrative tasks. Procedures are available in the ``system`` schema of the catalog.
```

Reply:
Thanks all for the suggestion, that looks wonderful, fixed.

Reply:
Looks good in a new build, thanks!